site/source/docs/optimizing/Profiling-Toolchain.rst

```diff
@@ -14,32 +14,30 @@ To try out the toolchain profiler, run the following set of commands:
 .. code-block:: bash
 
     cd path/to/emscripten
-    tools/emprofile.py --reset
-    export EM_PROFILE_TOOLCHAIN=1
+    export EMPROFILE=1
```

Comment:
Why change the name of this environment variable?

I myself only know of one user out there but there maybe be others.

Reply:
See https://github.com/emscripten-core/emscripten/pull/13464#issuecomment-777376747